packages/rum/test/e2e/react/webpack.config.js

```diff
@@ -37,9 +42,11 @@ module.exports = {
     rules: [
       {
         test: /.jsx?$/,
-        include: [__dirname, path.resolve(__dirname)],
         use: {
-          loader: 'babel-loader'
+          loader: 'babel-loader',
+          options: {
+            plugins: ['@babel/plugin-transform-destructuring']
```

Comment:
Plugin is not required. It is part of preset-env by default. https://github.com/babel/babel/blob/master/packages/babel-preset-env/src/available-plugins.js#L14

Reply:
The tests on Safari 9 fail without these because of bundling with unsupported ES6 features.

Reply:
Weird, because we do transform them to Es5 along with the plugins using our `.babelrc.js` in tests so I am not sure bundling would be the reason.

Reply:
The bundle actually was not transformed to ES5! The issue was with the generated bundle for this.

Reply:
Its still failing now right? Do you have any error related to this? Seems like a bundling problem if this is the case.

Reply:
The tests are passing now! 
The errors were happing in the Safari 9 on saucelabs. The bundles were still in ES6 code before this change.

Reply:
Can you move this plugin alone to `test/e2e/.babelrc.js`, To avoid the duplication of babel configs.